.buildkite/pipeline.yml

```diff
@@ -564,6 +564,9 @@ steps:
       - label: "end to end test"
         command: julia --project=calibration/test calibration/test/e2e_test.jl
         artifact_paths: "calibration_end_to_end_test/*"
+        soft_fail:
+          - exit_status: 1
```

Comment:
Exit status for agent lost (timeout) is -1

Reply:
The job is failing because `sacct` is erroring and crashing the job with exit status 1:
```
sacct: error: slurm_persist_conn_open_without_init: failed to open persistent connection to host:head1:6819: Connection refused
sacct: error: Sending PersistInit msg: Connection refused
sacct: error: Problem talking to the database: Connection refused
ERROR: LoadError: failed process: Process(`sacct --allocations -u esmbuild --starttime now-1hour -o Submit,Start -n`, ProcessExited(1)) [1]
```

Reply:
Ah, okay. Well, exit code 1 is the exit code that would get if the test failed with a real failure, so it is not particularly useful to filter out this specific exit code (unfortunately).